feat(curriculum): Add fCC Authors Page Project to JS Cert (#49655)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/63ff70d3ee568219332868f3.md

---
id: 63ff70d3ee568219332868f3
title: Step 5
challengeType: 0
dashedName: step-5
---

# --description--

> In order to start working with the data, you will need to use another `.then()` method.
> 
> Chain another `.then()` to the existing `.then()` syntax. This time, what you need to pass into the `.then()` as the parameter is `data`. For the callback, use a curly brace because you will have more than one expression in the arrow function. For the `data`, log it to the console, for now, and open the console tto see what it looks like.

Reviewer: ```suggestion
In order to start working with the data, you will need to use another `.then()` method.

Chain another `.then()` to the existing `.then()` method. This time, pass in `data` as the parameter for the callback function. For the callback, use a curly brace because you will have more than one expression. Within your callback function, log `data` to the console to see what it looks like.
```

---
id: 63ff70d3ee568219332868f3
title: Step 4
challengeType: 0
dashedName: step-4
---

# --description--

> Now that you've parsed the response into JSON with the `res`, you need to access the data with another `.then()` method. Populating the data we get on the UI starts from here.

Reviewer 2: I think this part could be a little bit shorter.  Maybe something like this

```suggestion
In order to start working with the data, you will need to chain another `.then()` method. 
```
